Makefile

```diff
@@ -0,0 +1,15 @@
+include common.mk
+MODULES=scripts test
+
+lint:
+	flake8 --ignore=E501,E722,E402 $(MODULES)
+
+requirements:
+	echo -n '' >| requirements.txt
+	virtualenv -p $(shell which python3) tmp-venv
+	./tmp-venv/bin/pip install -r requirements.txt.in
```

Comment:
Is the intention of this recipe to completely refresh requirements? If, instead, If the intention is to bump compatible requirements, the current requirements would need to be installed on the line above.